service/authentication_validation.go

```diff
@@ -32,6 +33,10 @@ func ValidateAuthenticationCreationRequest(auth *model.AuthenticationCreateReque
 	// capitalize it so it's always the same format.
 	auth.ResourceType = util.Capitalize(auth.ResourceType)
 
+	if err := ValidateAzureSubcriptionId(auth); err != nil{
+		return fmt.Errorf("Subscription ID is invalid")
```

Comment:
Error messages should not be capitalized

```suggestion
		return fmt.Errorf("subscription ID is invalid")
```